workflow-to-js-transformer/src/main/java/com/github/cafdataprocessing/workflow/transform/TransformerFunctions.java

```diff
@@ -32,7 +34,18 @@
 public class TransformerFunctions {
     
     public static final Logger LOG = LoggerFactory.getLogger(TransformerFunctions.class);
-    
+
+    /**
+     * Escape characters in the passed value that need to be escaped before being written to JavaScript.
+     * @param valueToEscape value that should have characters escaped
+     * @return escaped value
+     * @throws NullPointerException if {@code valueToEscape} is null
+     */
+    public static String escapeForJavaScript(final String valueToEscape) {
+        Objects.requireNonNull(valueToEscape);
+        return StringEscapeUtils.escapeEcmaScript(valueToEscape);
```

Comment:
This function appears to be deprecated from this module.  It there any reason it is being used rather than the Apache Commons Text version?

Reply:
It's not deprecated in 3.4 of Commons Lang that we're using (from caf-common parent). It's deprecated since 3.6.